### DISCUSSION TITLE
Moderation tools to move individual "answers" to be threaded "replies" in Discussions

### DISCUSSION THREAD

**Discussion by matthewfeickert** [community/community#6097] [Discussions]
Hi. In a similar, but different, vein to https://github.com/github/feedback/discussions/3200, it would be nice if there was some form of moderation tools that allowed repo admins to take individual "answers" to a GitHub Discussion and move them to be a threaded "reply" to another answer.

## Motivation and Use Case

The motivation for this is sometimes people post an "answer" that is really a reply to an existing thread of "replies". This then segments the conversation across multiple "answers" when it was really a singular thread. At the moment there is no way (that I am away for) for repo maintainers to be able to merge or move answers/replies.

For an example of such a situation see https://github.com/scikit-hep/pyhf/discussions/1619

**Comment by jasonvarga** [comment]
This would be wonderful. We are constantly running into users creating new answers rather than replying to an existing answer.

Saying "please reply instead of creating a new answer" is a little much. I'd love to be able to just move it. Otherwise our entire repo would be filled with these responses by me.
